packages/front-end/types/app-features.ts

```diff
@@ -105,4 +104,9 @@ export type AppFeatures = {
   "october16-ff": boolean;
   "general-dashboards": boolean;
   "incremental-refresh": boolean;
+  "top-banner": Record<string, unknown>;
+  "gbdemo-checkout-layout": boolean;
+  "post-stratification": boolean;
+  liebenowflag9: boolean;
```

Comment:
Remove this `liebenowflag9`.  The feature is archived (and I think it was added by accident in the first place).